public_html/index.html

```diff
@@ -57,13 +57,13 @@ <h2 class="highlight__item__title">Firefox Hardware Report</h2>
             </a>
           </li>
           <li class="highlight__item">
-            <a href="http://webassembly.org/" class="link">
+            <a href="https://research.mozilla.org/webassembly/" class="link">
               <div class="imageWrapper">
                 <img src="assets/img/tile-webassembly.jpg">
               </div>
               <figcaption>
-                <h2 class="highlight__item__title">WebAssembly Browser Preview</h2>
-                <p>Experimental support in multiple browsers</p>
+                <h2 class="highlight__item__title">WebAssembly is Here</h2>
```

Comment:
I wanted to double check with you, @ejregithub.

I intentionally did not capitalize `is` (per Chicago, AP, and NYT styles). But did you actually want that capitalized?